src/boot/efi/stub.c

```diff
@@ -267,6 +294,11 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table) {
                         credential_initrd = NULL;
                 }
 
+                if (global_credential_initrd) {
+                        FreePool(global_credential_initrd);
```

Comment:
We have `mfree` for this, you might wanna touch the other ones while you're at it.

Reply:
Done